cdap-master/src/test/java/co/cask/cdap/master/startup/ConfigurationCheckTest.java

```diff
@@ -62,4 +62,9 @@ public void invalidAuditTopicTest() {
   public void invalidLogPartition() {
     runConfigurationCheck(Constants.Logging.LOG_PUBLISH_PARTITION_KEY, "application1");
   }
+
+  @Test (expected = RuntimeException.class)
+  public void invalidHBaseDDLExecutorConfTest() {
```

Comment:
@chtyim Filed JIRA https://issues.cask.co/browse/CDAP-11727 for creating unit test for loading extensions present in the specified directory.